plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/product-details/block.json

```diff
@@ -4,7 +4,6 @@
 	"icon": "info",
 	"title": "Product Details",
 	"description": "Display a product's description, attributes, and reviews.",
-	"category": "woocommerce",
```

Comment:
In this case, the block.json isn't extended, so we have to decide if we should keep `woocommerce` or `woocommerce-product-elements`.